packages/block-library/src/list-item/hooks/use-enter.js

```diff
@@ -47,11 +44,7 @@ export default function useEnter( props ) {
 				}
 				// Here we are in top level list so we need to split.
 				const blockRootClientId = getBlockRootClientId( clientId );
-				const blockParents = getBlockParents( clientId );
```

Comment:
These changes are irrelevant and are a leftover from an iteration of the initial implementation. We don't need to get the block parents because if we cannot outdent, the block root client id is the one of the top level list.